pandas/tests/test_categorical.py

```diff
@@ -1410,10 +1410,8 @@ def test_searchsorted(self):
         self.assert_numpy_array_equal(res, chk)
 
         # As above, but with a sorter array to reorder an unsorted array
-        res = c2.searchsorted(['bread', 'eggs'], side='right',
-                              sorter=[0, 1, 2, 3, 5, 4])
-        chk = s2.searchsorted(['bread', 'eggs'], side='right',
-                              sorter=[0, 1, 2, 3, 5, 4])
+        res = c2.searchsorted(['bread', 'eggs'], 'right', [0, 1, 2, 3, 5, 4])
```

Comment:
I feel somewhat concerned about this.  Doesn't this make things more complicated?  If we're trying to discourage usage of the `sorter` argument, it seems little strange that we're giving the user so many ways to pass it in.